scripts/EN/strtab_script

```diff
@@ -1,849 +1,849 @@
+802: ¥"Daydream recollection¥"
+803: ¥"A piercing voice¥"
+804: ¥"The constant snow¥"
+805: ¥"On top of the desk¥"
+806: ¥"Harry steps back¥"
+807: ¥"Where did the culprit go?¥"
+808: ¥"Give me that handgun!¥"
+809: ¥"Suddenly, a memory springs up¥"
+810: ¥"The approaching monsters¥"
+811: ¥"A moment of hesitation¥"
+812: ¥"Leaving Silent Hill Town Center¥"
+813: ¥"Calming down…¥"
+814: ¥"The endless nightmare¥"
+815: ¥"Running toward the amusement park¥"
+816: ¥"Darkness¥"
+817: ¥"Dahlia and Alessa¥"
+818: ¥"……….¥"
+819: ¥"The phone call that won't go through¥"
+820: ¥"The snowy town¥"
+821: ¥"The empty cafe¥"
+822: ¥"The sleeping man¥"
+823: ¥"Cybil waits for the man to wake up¥"
+824: ¥"A little girl?¥"
+825: ¥"Cybil gives the man a handgun¥"
+826: ¥"Cybil wakes the man up¥"
+827: ¥"No way out¥"
+828: ¥"The small shadow off in the fog¥"
+829: ¥"Attacked by the ferocious monster¥"
+830: ¥"The monster attacks Cybil¥"
+831: ¥"Calming down¥"
+832: ¥"The monster just runs away¥"
+833: ¥"Searching for the Police Station¥"
+834: ¥"The word ¥"POLICE¥"¥"
+835: ¥"Inside the meeting room¥"
+836: ¥"The dangling telephone receiver¥"
+837: ¥"Leaving the Police Station¥"
+838: ¥"The presence of a person?¥"
+839: ¥"The deserted school¥"
+840: ¥"No turning back¥"
+841: ¥"A terrible headache¥"
+842: ¥"A noise¥"
+843: ¥"A melody on a piano¥"
+844: ¥"Music Room¥"
+845: ¥"Questioning Cheryl¥"
+846: ¥"Cheryl shakes her head back and forth¥"
+847: ¥"The school becomes enveloped in darkness¥"
+848: ¥"Down the pitch black hallway¥"
+849: ¥"A little girl's scream¥"
+850: ¥"Something is…¥"
+851: ¥"The small swaying figure¥"
+852: ¥"The ransacked cafe¥"
+853: ¥"The thing tangled around Cybil's feet¥"
+854: ¥"Cheryl shakes her head up and down¥"
+855: ¥"Walking toward the cafe¥"
+856: ¥"The sky became light again¥"
+857: ¥"The horrible sight inside the cafe¥"
+858: ¥"Some kind of clue…¥"
+859: ¥"The area around the monster's corpse¥"
+860: ¥"The continuously ringing bell¥"
+861: ¥"Dahlia Gillespie¥"
+862: ¥"Cheryl begins to speak¥"
+863: ¥"Obstinate Cheryl¥"
+864: ¥"Cybil raises her voice without realizing it¥"
+865: ¥"Cheryl's resolution¥"
+866: ¥"Cybil kindly speaks to Cheryl¥"
+867: ¥"The murmuring of water¥"
+868: ¥"The figure bobbing in the current¥"
+869: ¥"The drowning girl¥"
+870: ¥"Desperate Cheryl¥"
+871: ¥"Cybil slips¥"
+872: ¥"World of black¥"
+873: ¥"Elevator¥"
+874: ¥"The thing that Cybil was holding on to was…¥"
+875: ¥"A green jelly-like substance¥"
+876: ¥"The ever descending box¥"
+877: ¥"The drowning boy¥"
+878: ¥"The doll from that day¥"
+879: ¥"The B1 lamp¥"
+880: ¥"A single door¥"
+881: ¥"Cybil kicks the door over and over¥"
+882: ¥"The profile of the man visible in the flames¥"
+883: ¥"There is no mistake…it is Harry.¥"
+884: ¥"Cybil carries the girl¥"
+885: ¥"They must escape quickly¥"
+886: ¥"The question that grows within Cybil's heart¥"
+887: ¥"The white staircase¥"
+888: ¥"It is not Harry…¥"
+889: ¥"Seeking out the Police Station¥"
```

Comment:
The string did not correspond to the subtitle and part of it was in Russian (repeated from id: 833?)